### PR TITLE
Ability to configure energy delta decimals + idle status emission

### PR DESCRIPTION
**Pull request by krasnyt**
I'd like to add the ability to control decimals on the output energy delta, useful especially for low-energy devices. Then I also added an option to configure if the node should emit "idle" state events (most likely on incoming 0-power inputs).

### CODE REVIEW

**Review by xoseperez — CHANGES_REQUESTED** [review]
Check default value for energydecimals so it does not break existing flows.

File: power-monitor/power-monitor.html [hunk 1]
```diff
@@ -91,6 +105,8 @@
             this.startafter = this.startafter || 1;
             this.stopthreshold = this.stopthreshold || 0;
             this.stopafter = this.stopafter || 1;
+            this.energydecimals = this.energydecimals || 0;
```
[the remark this answers is not in the document]
Comment (krasnyt) [reply power-monitor/power-monitor.html]: Good point, fixed.

**Comment by krasnyt** [comment]
@xoseperez thx. what has to be done now in order to release a new version? I'm not really familiar with these processes.

**Comment by xoseperez** [comment]
Already pushed to npm repo, should be available soon

**Pull request merged**